scripts/release.sh

```diff
@@ -423,9 +423,9 @@ build_helm_deps() {
   fi
 
   echo "Updating helm chart dependencies ..."
-  # Helm chart directory path -- /scripts --> /chart
-  chart_dir="$SCRIPT_DIR/../chart"
-  dep_chart_dir="$chart_dir/charts"
+
+  # Helm chart directory path -- /scripts --> /chart (or /charts)
+  local dep_chart_dir="$HELM_CHART_DIR/charts"
```

Comment:
Maybe also skip this on error out if `build_deps` is set but `HELM_CHART_DIR` is not set?